code/datums/migrants/migrant_wave.dm

```diff
@@ -35,38 +35,34 @@
 		amount += roles[role_type]
 	return amount
 
-/datum/migrant_wave/refugee
+/datum/migrant_wave/adventurer
 	name = "Adventure Party"
-	downgrade_wave = /datum/migrant_wave/refugee_down_one
-	can_roll = FALSE
```

Comment:
This is currently defaulting to weight 100 as it has no weight set.
It needs a much lower weight, probably the same as zybantines or bandits.